util/prep-bazel-airgapped-build.sh

```diff
@@ -3,170 +3,78 @@
-  # https://bazel.build/run/build#repository_cache_with_bazel_7_or_later
-  mkdir -p "${BAZEL_AIRGAPPED_DIR}/empty_workspace"
-  pushd "${BAZEL_AIRGAPPED_DIR}/empty_workspace"
-    touch MODULE.bazel
-    cp "${REPO_TOP}/.bazelversion" .
-    bazel fetch --repository_cache="${BAZEL_AIRGAPPED_DIR}/${BAZEL_CACHEDIR}"
-  popd
-  rm -rf "${BAZEL_AIRGAPPED_DIR}/empty_workspace"
-fi
-
-################################################################################
-# Prepare the cache.
-################################################################################
-if [[ ${AIRGAPPED_DIR_CONTENTS} == "ALL" || \
-      ${AIRGAPPED_DIR_CONTENTS} == "CACHE" ]]; then
-  echo $LINE_SEP
-  echo "Preparing bazel offline cachedir ..."
-  cd ${REPO_TOP}
-  mkdir -p ${BAZEL_AIRGAPPED_DIR}/${BAZEL_CACHEDIR}
-  # Make bazel forget everything it knows, then download everything.
-  ${BAZELISK} clean --expunge
-  ${BAZELISK} fetch \
-    --repository_cache=${BAZEL_AIRGAPPED_DIR}/${BAZEL_CACHEDIR} \
-    //... \
-    @lowrisc_rv32imcb_toolchain//... \
-    @local_config_platform//... \
-    @riscv-compliance//... \
-    @rules_foreign_cc//toolchains/... \
-  # We don't need all bitstreams in the cache, we just need the latest one so
-  # that the cache is "initialized" and "offline" mode will work correctly.
-  mkdir -p ${BAZEL_AIRGAPPED_DIR}/${BAZEL_BITSTREAMS_CACHEDIR}
-  readonly SYSTEM_BITSTREAM_CACHE="${HOME}/.cache/opentitan-bitstreams"
-  readonly SYSTEM_BITSTREAM_CACHEDIR="${SYSTEM_BITSTREAM_CACHE}/cache"
-  readonly LATEST_BISTREAM_HASH_FILE="${SYSTEM_BITSTREAM_CACHE}/latest.txt"
-  # The revision named in latest.txt is not necessarily on disk. Induce the
-  # cache backend to fetch the latest bitstreams.
-  BITSTREAM=latest ${BAZELISK} fetch @bitstreams//...
-  cp "${LATEST_BISTREAM_HASH_FILE}" \
-    "${BAZEL_AIRGAPPED_DIR}/${BAZEL_BITSTREAMS_CACHE}/"
-  LATEST_BISTREAM_HASH=$(cat "${LATEST_BISTREAM_HASH_FILE}")
-  cp -r "${SYSTEM_BITSTREAM_CACHEDIR}/${LATEST_BISTREAM_HASH}" \
-    "${BAZEL_AIRGAPPED_DIR}/${BAZEL_BITSTREAMS_CACHEDIR}"
-  echo "Done."
-fi
-
-################################################################################
-# Print some usage instructions.
-################################################################################
-if [[ ${AIRGAPPED_DIR_CONTENTS} == "ALL" ]]; then
-  echo $LINE_SEP
-  echo "To perform an airgapped build, ship the contents of ${BAZEL_AIRGAPPED_DIR} to your airgapped environment and then:"
-  echo ""
-  echo "bazel build --distdir=${BAZEL_AIRGAPPED_DIR}/${BAZEL_DISTDIR} --repository_cache=${BAZEL_AIRGAPPED_DIR}/${BAZEL_CACHEDIR} <label>"
-fi
+fi  
+
+# Create directories
+mkdir -p "$BAZEL_AIRGAPPED_DIR"
+mkdir -p "$BAZEL_BITSTREAM_DIR"
+mkdir -p "$BAZEL_VENDOR_DIR"
+
+echo "Downloading Bazel binary..."
+
+curl --silent --location \
+  "https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-linux-x86_64" \
+  --output "${BAZEL_AIRGAPPED_DIR}/bazel"
```

Comment:
Maybe `bazelisk` should do this download so it can check the hash and we don't have to read `.bazelversion` ourselves.